### PR TITLE
feat: Add expense claim qualification for invoice, finance and work&study

### PR DESCRIPTION
**Pull request by JF-Cozy**

### CODE REVIEW

**Review by zatteo — APPROVED** [review]

**Review by paultranvan — CHANGES_REQUESTED** [review]
It looks like you want to add a qualification, but this qualification is never actually added. The `expense_claim` should be added in the qualifications.json doc : https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/assets/qualifications.json

**Comment by JF-Cozy** [comment]
merci @paultranvan c'est corrigé

File: packages/cozy-client/src/assets/qualifications.json [hunk 1]
```diff
@@ -491,6 +491,9 @@
     {
       "label": "other_administrative_document"
     },
+    {
+      "label": "expense_claim"
+    },
```
Comment (paultranvan) [inline packages/cozy-client/src/assets/qualifications.json:496]: Would you mind just adding the purpose? :pray: 
```
{
  "label": "expense_claim"
  "purpose": "invoice"  
},
```

**Review by paultranvan — APPROVED** [review]

File: packages/cozy-client/src/assets/qualifications.json (hunk 1 above)
Comment (JF-Cozy) [reply packages/cozy-client/src/assets/qualifications.json:496]: pas de souci, avec @BenjaminMty on ne savait pas trop quoi mettre 👍

**Pull request merged**